virttest/data_dir.py

```diff
@@ -201,6 +201,10 @@ def get_deps_dir(target=None):
 def get_tmp_dir():
     return data_dir.get_tmp_dir()
 
+def get_public_tmp_dir():
+    tmp_dir = data_dir.get_tmp_dir()
+    os.chmod(tmp_dir, 0755)
+    return tmp_dir
 
```

Comment:
In my humble opinion, 
There is no need to add a function in case of get_tmp_dir has similar functions with it.
We can add new param to allows users to set permissions or select which level they need.